CHANGELOG.md

```diff
@@ -120,6 +120,8 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 - server: accept only non-negative integers for batch size and refetch interval (close #5653) (#5759)
 - server: fix bug which arised when renaming a table which had a manual relationship defined (close #4158)
 - server: limit the length of event trigger names (close #5786)
+- server: Configurable websocket keep-alive interval. Add `--websockets-keepalive` command-line flag
```

Comment:
Can we call it `--websocket-keepalive` as it is more natural and consistent with the ENV var?